src/apis/airdrop.js

```diff
@@ -0,0 +1,28 @@
+import axios from 'axios'
+
+// import { HERMEZ_WEB_URL } from '../constants'
+
+// const baseApiUrl = HERMEZ_WEB_URL
+
+// TODO: change back proper hermez web URL
+const baseApiUrl = 'http://52.29.96.190'
```

Comment:
I'll add this as a constant as in the comments above. Also I think that the `baseApiUrl` should be `http://52.29.96.190/airdrop/v1`.